scripts/rabbit_dependencies.sh

```diff
@@ -14,8 +14,8 @@ function exit_on_error {
 function print_usage {
  echo "
 Command Usage:
-<path>/rabbit_dependencies.sh <ubuntu or centos> <distribution name or centos version>
-Valid Ubuntu distributions: ${list[@]}
+<path>/rabbit_dependencies.sh <debian or centos> <distribution name or centos version>
+Valid Debian distributions: ${list[@]}
```

Comment:
should be debian based....ubuntu is debian based...

Reply:
Fixed